### ISSUE TITLE
Perform Copy Number Variation analysis on C gigas samples

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#1656]
for @ggoetznoaa / @mgavery can advise

TLDR: [Nightingales Folder](http://owl.fish.washington.edu/nightingales/C_gigas/) - F05/F14 prefix <- raw data

Short-read WGS, 30x coverage, 32 samples, 2 families, 8 samples per ploidy/family. See [notebook post](https://mattgeorgephd.github.io/USDA-NRSP-8-gigas-rDNA-analysis-2/) for more information.

WGS results received from Azenta. Stored on nightingales. Intial QC and trimming done on Raven. See [notebook post](https://mattgeorgephd.github.io/USDA-NRSP-8-gigas-rDNA-analysis-2/).

---

## Pertinent documents

[repo](https://github.com/mattgeorgephd/USDA-NRSP-8-gigas-rDNA)

[Manuscript](https://github.com/mattgeorgephd/USDA-NRSP-8-gigas-rDNA/blob/main)
[Proposal](https://github.com/mattgeorgephd/USDA-NRSP-8-gigas-rDNA/blob/939c3271038ea9007e09eb1bd4ca4151e1636aad/proposal/George-NRSP8_2022.pdf)
[Tissue Sample List](https://github.com/mattgeorgephd/USDA-NRSP-8-gigas-rDNA/blob/f83756e93d4fb7ebf40c31b2095f9e4749f7cff3/morphometrics/morphometrics.xlsx)
https://github.com/RobertsLab/resources/issues/1304
[DNA extraction protocol](https://docs.google.com/document/d/1_qi5NZfPaGzD9gh-u01s8uGYy2OMPolZeltc9XMIE9M/edit?usp=sharing)
[DNA extraction results](https://docs.google.com/spreadsheets/d/1mT-RngtGx3yUCDWR5lFlwokGhHqkjkrOQ7xhuaMOPes/edit?usp=sharing)
[Gannet Folder](https://gannet.fish.washington.edu/panopea/USDA-NRSP-8-gigas-rDNA/)
[Nightingales Folder](http://owl.fish.washington.edu/nightingales/C_gigas/) - F05/F14 prefix <- *raw data*

**Comment by ggoetznoaa** [comment]
@sr320 I see the files but I can't download them. It just sorta hangs when I try either via the web browser or via command line program. Do I need a login/password?

**Comment by ggoetznoaa** [comment]
@sr320 oh and I never got an email saying you tagged me. Not sure if its something on my end that needs to be changed or your end. I usually don't use github for this sort of stuff.

**Comment by kubu4** [comment]
Hmmm, sorry you're having trouble! All the files are publicly accessible, so no need for a password.

I just tested downloading a file and proceeded without issue:

`wget https://owl.fish.washington.edu/nightingales/C_gigas/F142n01_R1_001.fastq.gz`

Obviously, this isn't too much help, since it doesn't solve your problem...

**Comment by ggoetznoaa** [comment]
it could be something on NOAA's side, firewall setting etc. I was able to get your command to run but when I tried the following it failed.

wget http://owl.fish.washington.edu/nightingales/C_gigas/0501_R1_001.fastq.gz

**Comment by ggoetznoaa** [comment]
Ah I think I see the issue. The link in the doc doesn't have https and firefox went with http and that doesn't seem to be working.

**Comment by ggoetznoaa** [comment]
yep its working, as soon as I added https.

**Comment by ggoetznoaa** [comment]
Ok I've downloaded the files. I'm assuming the files I've downloaded are the raw files and not the trimmed/QC'd files Matt mentions in his lab doc. I see all 32 samples (16 F0 and 16 F14).

**Comment by sr320** [comment]
@ggoetznoaa  just checking in to see where this is at?

**Comment by mgavery** [comment]
It's almost ready to move to my plate! Giles has done the mapping and pulled coverage for single copy genes, mito and ribo using bedtools.  He is going to make tables that consolidate coverage for all samples into a single table and then I'll analyze in R.

**Comment by sr320** [comment]
Can I get access to bam files?

**Comment by ggoetznoaa** [comment]
@mgavery yep, just finished making the three different table files.

@sr320 I still have the BAM files, there are 32 files totaling 172 GB. I can put them up in a google drive for you to download them from unless you have another place you want me to put them.

**Comment by sr320** [comment]
@ggoetznoaa can you put on mox scrubbed directory?

**Comment by ggoetznoaa** [comment]
@sr320 I have no idea what that is or where that is. Is it a server you have? if so I don't think I have access to it.

**Comment by sr320** [comment]
mox - hyak..... mox.hyak.uw.edu ; you have account. 

other suggestion is fine but google docs?? cannot do anything with files that big in google docs.

**Comment by mgavery** [comment]
@sr320 I have an acct on gannet.  I can put them there.  Does that work?

**Comment by sr320** [comment]
@mgavery yes that would be great. thanks!

**Comment by mgavery** [comment]
<img width="791" alt="Notification_Center" src="https://github.com/RobertsLab/resources/assets/4512578/c382f744-1663-401b-aafb-e416c44fbe5a">

**Comment by mgavery** [comment]
First pass at looking at coverage per sample for ~5000 single copy genes (top left) and then average copy number of mito and ribo genes

**Comment by mattgeorgephd** [comment]
Amazing the variation within family/ploidy. Some oysters have x3!

**Comment by mgavery** [comment]
[Dip_v_trip_mito_ribo_copynum.csv](https://github.com/RobertsLab/resources/files/12363172/Dip_v_trip_mito_ribo_copynum.csv)

This is the file the plots were generated from.  I haven't done any stats yet

**Comment by sr320** [comment]
Here is what I have for global analysis... https://rpubs.com/sr320/1070681
